config/crd/bases/manila.openstack.org_manilas.yaml

```diff
@@ -672,6 +672,11 @@ spec:
                       caBundleSecretName:
                         type: string
                     type: object
+                  workers:
+                    default: 3
```

Comment:
@gouthampacha is this a good "default" ? Do we want to stay w/ `4` (the original hardcoded value), or do we want something else?